### PR TITLE
Added TensorFlow Linalg eigvals function and tests.

### PR DESCRIPTION
**Pull request by Harshalzzzzzzz**
#4794

### CODE REVIEW

**Comment by Harshalzzzzzzz** [comment]
@sreegandh804 Can you please review this PR. I will update the PR with suggested changes and inputs asap.
Thanks

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hey! 
First of all thank you soo much for contributing to ivy! 🙂
Your PR looks amazing ✨!
Just requested a change and left a comment on it which needs to be addressed before we can merge your PR! 🙂
Thanks!

File: ivy/functional/frontends/tensorflow/linalg.py [hunk 1]
```diff
@@ -10,6 +10,10 @@ def det(input, name=None):
     return ivy.det(input)
 
 
+def eigvals(tensor, name=None):
+    return ivy.eigvals(tensor)
```
Comment (zaeemansari70) [inline ivy/functional/frontends/tensorflow/linalg.py:14]: This function is not present in `Ivy`. It is present in our Extended API task.
You can work on it to implement it.
Thanks!
https://github.com/unifyai/ivy/issues/3856

**Comment by fspyridakos** [comment]
Due to lack of activity, I will be closing this PR.

**Pull request closed without merging**